supervisor/addons/validate.py

```diff
@@ -284,6 +287,9 @@ def _migrate(config: Dict[str, Any]):
         vol.Optional(ATTR_SNAPSHOT_EXCLUDE): [str],
         vol.Optional(ATTR_SNAPSHOT_PRE): str,
         vol.Optional(ATTR_SNAPSHOT_POST): str,
+        vol.Optional(ATTR_SNAPSHOT, default=SnapshotAddonMode.HOT): vol.Coerce(
```

Comment:
```suggestion
        vol.Optional(ATTR_SNAPSHOT, default=SnapshotAddonMode.HOT): vol.Coerce(
```
This will be an external-facing entry, we should use `ATTR_BACKUP` here

Reply:
The API creates a backup but the system is a snapshot. I see no reason to reflect a UI decision into the backend. I'm also fine with the API because it's the overlay, but then it really stop on what I will rename

Reply:
Sure, but this is part of the add-on configuration, that is user-facing and should be the same.
We should make sure we use the same naming for things for everything that is exposed outside, or not change it at all.

Reply:
I agree with @ludeeus. Sure its more painful now, but it will be confusing in the future otherwise.